pyproject.toml

```
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "epcsunspecdemo"
+version = "0.0.0"
+description = ""
+authors = ["Alex Anker <alex.anker@epcpower.com>"]
+license = "MIT"
+
+[tool.poetry-dynamic-versioning]
+enable = true
+vcs = "git"
+
+[tool.poetry-dynamic-versioning.substitution]
+files = ["src/epcsunspecdemo/__init__.py"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+attrs = "18.2.0"
+click = "7"
+toolz = "0.9.0"
+tqdm = "4.32.1"
+pysunspec = "2.1.1"
+setuptools = "53.0.0"
+toml = "^0.10.2"
+importlib = "^1.0.4"
+importlib-metadata = "^4.10.1"
+typing-extensions = "^4.0.1"
+[tool.poetry.dev-dependencies]
```

Nitpick add empty line.
```suggestion
typing-extensions = "^4.0.1"

[tool.poetry.dev-dependencies]
```